### ISSUE TITLE
site-packages/torch/distributed/elastic/multiprocessing/api.py:869] failed (exitcode: -8) local_rank: 0

### ISSUE THREAD

**Issue by Helioswei** [hpcaitech/Open-Sora#872]
=======================
torchrun --nproc_per_node 1 --standalone scripts/diffusion/inference.py configs/diffusion/inference/256px.py --cond_type i2v_head --prompt "A plump pig wallows in a muddy pond on a rustic farm, its pink snout poking out as it snorts contentedly. The camera captures the pig's playful splashes, sending ripples through the water under the midday sun. Wooden fences and a red barn stand in the background, framed by rolling green hills. The pig's muddy coat glistens in the sunlight, showcasing the simple pleasures of its carefree life." --ref assets/texts/i2v.png

[06/22/25 19:50:39] INFO     colossalai - colossalai - INFO:                                                               
                             /usr/local/lib/python3.10/site-packages/colossalai/initialize.py:75 launch                    
                    INFO     colossalai - colossalai - INFO: Distributed environment is initialized, world size: 1         
[2025-06-22 19:50:39] Inference configuration:
 {'ae': {'from_pretrained': './ckpts/hunyuan_vae.safetensors',
        'in_channels': 3,
        'latent_channels': 16,
        'layers_per_block': 2,
        'out_channels': 3,
        'type': 'hunyuan_vae',
        'use_spatial_tiling': True,
        'use_temporal_tiling': False},
 'batch_size': 1,
 'clip': {'from_pretrained': './ckpts/openai/clip-vit-large-patch14',
          'max_length': 77,
          'type': 'text_embedder'},
 'cond_type': 'i2v_head',
 'config_path': 'configs/diffusion/inference/256px.py',
 'dataset': {'type': 'text'},
 'dtype': 'bf16',
 'fps_save': 24,
 'model': {'axes_dim': [16, 56, 56],
           'cond_embed': True,
           'context_in_dim': 4096,
           'depth': 19,
           'depth_single_blocks': 38,
           'from_pretrained': './ckpts/Open_Sora_v2.safetensors',
           'fused_qkv': False,
           'guidance_embed': False,
           'hidden_size': 3072,
           'in_channels': 64,
           'mlp_ratio': 4.0,
           'num_heads': 24,
           'qkv_bias': True,
           'theta': 10000,
           'type': 'flux',
           'use_liger_rope': True,
           'vec_in_dim': 768},
 'motion_score': '4',
 'prompt': 'A plump pig wallows in a muddy pond on a rustic farm, its pink '
           'snout poking out as it snorts contentedly. The camera captures the '
           "pig's playful splashes, sending ripples through the water under "
           'the midday sun. Wooden fences and a red barn stand in the '
           "background, framed by rolling green hills. The pig's muddy coat "
           'glistens in the sunlight, showcasing the simple pleasures of its '
           'carefree life.',
 'ref': 'assets/texts/i2v.png',
 'sampling_option': {'aspect_ratio': '16:9',
                     'guidance': 7.5,
                     'guidance_img': 3.0,
                     'image_osci': True,
                     'is_causal_vae': True,
                     'method': 'i2v',
                     'num_frames': 129,
                     'num_steps': 50,
                     'resolution': '256px',
                     'scale_temporal_osci': True,
                     'seed': None,
                     'shift': True,
                     'temporal_reduction': 4,
                     'text_osci': True},
 'save_dir': 'samples',
 'seed': 42,
 't5': {'from_pretrained': './ckpts/google/t5-v1_1-xxl',
        'max_length': 512,
        'shardformer': True,
        'type': 'text_embedder'}}
[2025-06-22 19:50:39] Building dataset...
[rank0]:[W622 19:50:39.175010322 ProcessGroupNCCL.cpp:4115] [PG ID 0 PG GUID 0 Rank 0]  using GPU 0 to perform barrier as devices used by this process are currently unknown. This can potentially cause a hang if this rank to GPU mapping is incorrect.Specify device_ids in barrier() to force use of a particular device,or call init_process_group() with a device_id.
[2025-06-22 19:50:39] Dataset contains 1 samples.
[2025-06-22 19:50:39] Building models...
[2025-06-22 19:50:39] Loading checkpoint from ./ckpts/Open_Sora_v2.safetensors
[2025-06-22 19:50:42] Model loaded successfully
[2025-06-22 19:50:42] Loading checkpoint from ./ckpts/hunyuan_vae.safetensors
[2025-06-22 19:50:43] Model loaded successfully
Loading checkpoint shards: 100%|█████████████████████████████████████████████████████████████| 2/2 [00:13<00:00,  6.57s/it]
[2025-06-22 19:50:57] CUDA max memory max memory allocated at build model: 31.7 GB
[2025-06-22 19:50:57] CUDA max memory max memory reserved at build model: 32.2 GB
Inference progress:   0%|                                                                            | 0/1 [00:00<?, ?it/s][2025-06-22 19:50:57] Generating video...
E0622 19:51:03.299000 34886 site-packages/torch/distributed/elastic/multiprocessing/api.py:869] failed (exitcode: -8) local_rank: 0 (pid: 34962) of binary: /usr/local/bin/python3.10

how to fix this error?
| NVIDIA-SMI 535.216.01             Driver Version: 535.216.01   CUDA Version: 12.2     |
python -c "import torch; print(torch.__version__); print(torch.version.cuda)"
2.5.1+cu121
12.1

**Comment by FunTotal** [comment]
Meet the same problem, do you solve this problem?
